data/algae/algae_cube/0.mtl

```diff
@@ -0,0 +1,13 @@
+# Blender MTL File: 'algae.blend'
+# Material Count: 1
+
+newmtl algae_green
+Ns 225.000000
+Ka 1.000000 1.000000 1.000000
+Kd 0.800000 0.800000 0.800000
+Ks 0.500000 0.500000 0.500000
+Ke 0.000000 0.000000 0.000000
+Ni 1.450000
+d 1.000000
+illum 2
+map_Kd /home/li/Desktop/smarc/catkin_ws/src/smarc_stonefish_sims/data/algae/textures/algae_green.jpg
```

Comment:
I don't think stonefish can use .mtl files. These things should probably be within the scenario files instead.